data/boosters/clb-collector.yaml

```diff
@@ -23,11 +23,11 @@ sheets:
     use: common_uncommon
     count: 50
   extended_commander:
-    filter: "e:clb number:607-645"
+    filter: "e:clb number:607-645 -is:foil"
```

Comment:
Change this line to: 
`filter: "e:clb number:607-645 -alt:(e:clb number:646-657)"`